tests/checks/mock/test_splunk_topology.py

```diff
@@ -0,0 +1,362 @@
+# stdlib
```

Comment:
Nice Tests! Perhaps we can add test which verify the scenario when an error response is returned by splunk.